docs/features/sessions/README.md

```diff
@@ -48,4 +48,21 @@ The equivalent CLI command:
 ```text
 grid session delete $SESSION_NAME
 ```
+## Change Instance Type of a Session
+
+Change instance type allows you to upgrade or downgrade the compute capability of the session node while keeping all of your work in progress untouched. It is particularly useful if you want to increase/decrease the resource capacity of an existing session.
+
+![](/images/sessions/change-instance-type.gif)
+
+You can change the instance type of the session when it is PAUSED
```

Comment:
(Test this) When using the CLI, you can only change the instance type when the Session is paused.